iminuit/_libiminuit.pyx

```diff
@@ -78,30 +75,69 @@ cdef class Minuit:
     #: raise runtime error if function evaluate to nan
     cdef readonly bint throw_nan
 
-    #PyMinuit Compatible interface
-    cdef readonly object parameters#:tuple of parameter name(correct order)
-    cdef readonly object args#:tuple of values
-    cdef readonly object values#:map varname -> value
-    cdef readonly object errors#:map varname -> parabolic error
-    cdef readonly object covariance#:map (v1,v2)->covariance
-    cdef readonly double fval#:last value of fcn
+    # PyMinuit Compatible interface
+    cdef readonly object parameters
+    """Parameter name tuple"""
+
+    cdef readonly object args
+    """Parameter value tuple"""
+
+    cdef readonly object values
+    """Parameter values (dict: name -> value)"""
+
+    cdef readonly object errors
+    """Parameter parabolic errors (dict: name -> error)"""
+
+    cdef readonly object covariance
+    """Covariance matrix (dict (name1, name2) -> covariance)"""
+
+    cdef readonly double fval
+    """Last evaluated FCN value
+
+    .. seealso:: :meth:`get_fmin`
+    """
+
     cdef readonly int ncalls#:number of fcn call of last migrad/minos/hesse
     cdef readonly double edm#Estimate distance to minimum
     #:minos error ni a funny map from
     #:(vname,direction)->error
     #:direction is 1.0 for positive error and -1.0 for negative error
+
     cdef readonly object merrors
+    """MINOS errors (dict).
+
+    Using this method is not recommended.
+    It was added only for PyMinuit compatibility.
+    Use :meth:`get_merrors` instead, which returns a dictionary of
+    name -> :ref:`minos-error-struct` instead.
+
+    Dictionary entries for each parameter:
+
+    * (name,1.0) -> upper error
+    * (name,-1.0) -> lower error
+    """
+
     #:global correlation coefficient
     cdef readonly object gcc
-    #and some extra
-    #:map of
-    #:varname -> value
-    #:error_varname -> error
-    #:limit_varname -> limit
-    #:fix_varname -> True/False
-    #:user can just use python keyword expansion to use all the argument like
-    #:Minuit(fcn,**fitargs)
+
     cdef public object fitarg
+    """Current Minuit state in form of a dict.
```

Comment:
My guess # cython: embedsignature=True, at the top of the file. You want it though.